jobs/pacman-qualys-enricher/src/main/java/com/tmobile/cso/pacman/qualys/util/ErrorManageUtil.java

```diff
@@ -32,10 +32,11 @@ public static Map<String,Object> formErrorCode(List<Map<String,String>> errorLis
             for(Map<String, String> errorDetail :errorList) {
                 Map<String,Object> error = new HashMap<>();
                 error.put(ERROR, errorDetail.get(ERROR));
-                
+                log.error("error detail - {}", errorDetail.get(ERROR));
```

Comment:
Same here. To many log statements. Let's combine into a single log statement for each error

Reply:
done.